lib/sisimai/bite/email/interscanmss.rb

```diff
@@ -44,9 +32,17 @@ def scan(mhead, mbody)
         return nil unless mhead
         return nil unless mbody
 
-        match  = 0
-        match += 1 if mhead['from'] =~ Re0[:from]
-        match += 1 if Re0[:subject].find { |a| mhead['subject'] == a }
+        # :received => %r/[ ][(]InterScanMSS[)][ ]with[ ]/,
+        match = 0
+        tryto = [
+          'Mail could not be delivered',
+          # メッセージを配信できません。
```

Comment:
__[RuboCop]__

Style/AsciiComments: Use only ascii symbols in comments.


<!-- Comment by [SideCI](https://sideci.com) -->